content/fan-noise.md

```diff
@@ -0,0 +1,36 @@
+---
+title: System Fan Noise and Expectactions
+description: >
+  Ways to manage fan noise and design constraints.
+keywords:
+  - Fans
+  - Loud
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+## Noise Levels
+
+System76 sets custom fan curves for laptops to maximize their performance while minimazing fan noise. Current firmware keeps the fan off until 65c and will ramp the fan speeds to max speed by 90c.
+
+### Monitoring Fan Speeds
+
+Fan Speeds can be monitored with <u>psensor</u>, which can be installed from either the <u>Pop!_Shop</u> or the <u>Ubuntu Software</u> applications. Below is an example from a Oryx Pro (the number of cores/threads/GPU presense will be different depending on the system).
+
+![psensor on an oryp7](/images/fan-noise/psensor-oryp7.png)
+
+### Fan Modes
+
+Our Open Firmware laptops (so other than the Pangolin) have fan controls to swap between max or automatic thermal control. Use Fn+1 to toggle between the modes. (Your model will show support on the key section of their [tech-docs](https://tech-docs.system76.com)).
```

Comment:
>(so other than the Pangolin)

This doesn't work for me. Older laptops also don't have open firmware. If you want to be specific, it'd probably be best to link to https://github.com/system76/firmware-open/blob/master/FEATURES.md or something. Otherwise, just delete this part. 

>will show support on the key section

Be more specific than "the key section". Something like `the "Keyboard Shortcuts" section in the External Overview` would be much more specific.